apps/rush-lib/src/cli/actions/CustomRushAction.ts

```diff
@@ -148,6 +155,13 @@ export class CustomRushAction extends BaseRushAction {
       parameterShortName: '-v',
       description: 'Display the logs during the build, rather than just displaying the build status summary'
     });
+    if (this.options.actionVerb === 'build') {
+      this._ignoreDownstream = this.defineFlagParameter({
+        parameterLongName: '--no-deps',
+        description: 'If specified, the incremental build will only rebuild projects that have changed, '
+          + 'but not their dependents.'
```

Comment:
"dependent" is an unclear word because it's uncommon.  It's very useful to define this jargon and use it consistently everywhere in our code, but the end-user documentation should try to avoid jargon.

Maybe say "but not other projects that depend on them"?

Reply:
What is the effect if you specify `-t` or `-f`?

Reply:
It will only rebuild projects that have changed if they are in the dependency graph 

---
In reply to: [164222227](https://github.com/Microsoft/web-build-tools/pull/514#discussion_r164222227) [](ancestors = 164222227)